crates/uv/src/commands/pip_install.rs

```diff
@@ -237,6 +238,35 @@ pub(crate) async fn pip_install(
         Err(err) => return Err(err.into()),
     };
 
+    if dry_run {
+        writeln!(
+            printer,
+            "{}",
+            format!(
+                "Would install {}",
+                format!("{} packages", resolution.len()).bold(),
+            )
+            .dimmed()
+        )?;
+
+        for package_name in resolution.packages() {
+            if let Some(dist) = resolution.get(package_name) {
+                let version = dist
+                    .version()
+                    .map_or_else(String::new, |version| format!("=={version}"));
+
+                writeln!(
+                    printer,
+                    " {} {}{}",
+                    "~".blue(),
```

Comment:
```suggestion
                    "+".blue(),
```
Not sure how this should look? `+` and `-` didn't seem to fit, though.. so I tried `~`